actions/admin/security/securitytxt.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Save the configuration of the security.txt contents
+ */
+
+$fields = [
+	'contact' => true,
+	'expires' => true,
+	'encryption' => false,
+	'acknowledgments' => false,
+	'language' => false,
+	'canonical' => false,
+	'policy' => false,
+	'hiring' => false,
+	'csaf' => false,
+];
+
+foreach ($fields as $name => $required) {
+	$value = get_input($name);
+	if ($required && empty($value)) {
+		return elgg_error_response(elgg_echo('error:missing_data'));
+	}
+	
+	elgg_save_config("securitytxt_{$name}", $value);
```

Comment:
can you first validate all fields before saving them?

Reply:
done